.github/workflows/CI-build.yml

```
@@ -0,0 +1,32 @@
+name: Build
+
+run-name: "Workflow CI/CD Step: cargo build"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: warp-ubuntu-latest-x64-8x
+    name: Build evm-paranode
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Evm-paranode build
+        uses: ./.github/actions/cmd-in-docker
+        with:
+          command: "cargo build -p zkv-para-evm-node --release"
+          use_cache: 'yes'
+          cache_key: 'build-evm-node'
+          lld_install: 'yes'
+
+      - name: Upload evm-paranode build binaries
```

paranode just sound a bit like `paranoid` :)
```suggestion
      - name: Upload para-evm-node build binaries
```